mountpoint-s3-crt-sys/Cargo.toml

```diff
@@ -8,38 +8,56 @@ repository = "https://github.com/awslabs/mountpoint-s3"
 description = "Rust FFI bindings to the AWS Common Runtime for Mountpoint for Amazon S3."
 exclude = [
     # Exclude large files/directories not required to build the CRT (e.g. tests, docs)
+    "crt/*/samples/**",
     "crt/*/tests",
     "!crt/aws-lc/tests/compiler_features_tests",
     "!crt/s2n-tls/tests/features",
+    "crt/*/verification",
+    "crt/*/include/aws/testing",
     "crt/**/docs",
     "crt/aws-c-cal/ecdsa-fuzz-corpus/*",
     "crt/aws-c-common/verification/*",
-    "crt/aws-lc/crypto/*/*_tests.txt",
+    "crt/aws-c-common/AWSCRTAndroidTestRunner",
+    "crt/aws-c-s3/benchmarks",
+    "crt/aws-lc/**/*test*.go",
+    "crt/aws-lc/**/*test*.json",
+    "crt/aws-lc/**/*test*.py",
+    "crt/aws-lc/**/*test*.txt",
+    "crt/aws-lc/crypto/**/*_test.cc",
     "crt/aws-lc/crypto/cipher_extra/test/*",
-    "crt/aws-lc/crypto/evp_extra/*_test.cc",
-    "crt/aws-lc/crypto/evp_extra/*_tests.txt",
     "crt/aws-lc/crypto/fipsmodule/bn/test/*",
-    "crt/aws-lc/crypto/fipsmodule/*/*_tests.txt",
     "crt/aws-lc/crypto/fipsmodule/policydocs/*",
     "crt/aws-lc/crypto/fipsmodule/sha/testvectors/*",
-    "crt/aws-lc/crypto/hpke/hpke_test_vectors.txt",
-    "crt/aws-lc/crypto/hpke/test-vectors.json",
     "crt/aws-lc/crypto/kyber/kat/*",
-    "crt/aws-lc/crypto/x509/*_test.cc",
     "crt/aws-lc/crypto/x509/test/*",
+    "crt/aws-lc/crypto/ocsp/test/*",
     "crt/aws-lc/fuzz/*",
     "crt/aws-lc/generated-src/crypto_test_data.cc",
-    "crt/aws-lc/ssl/ssl_test.cc",
-    "crt/aws-lc/ssl/test/*",
-    "crt/aws-lc/third_party/googletest/test/*",
+    "crt/aws-lc/ssl",
```

Comment:
Is this okay because we're using s2n-tls?

Reply:
I'm not 100% sure, so I've reverted this line in particular. On Linux, I believe so, yes - but it's unclear to me what would happen on other OS.

Reply:
When excluding `crt/aws-lc/ssl/`:

```
Run cargo package -p mountpoint-s3-crt-sys --no-verify
    Updating crates.io index
   Packaging mountpoint-s3-crt-sys v0.8.0 (/home/runner/work/mountpoint-s3/mountpoint-s3/mountpoint-s3-crt-sys)
    Packaged 2139 files, 44.4MiB (8.3MiB compressed)
```

When not excluding `crt/aws-lc/ssl/`:

```
❯ cargo package -p mountpoint-s3-crt-sys --no-verify --allow-dirty
   Packaging mountpoint-s3-crt-sys v0.8.0 (/Users/djonesoa/devel/mountpoint-s3/mountpoint-s3-crt-sys)
    Packaged 2185 files, 45.7MiB (8.6MiB compressed)
```

Reply:
Okay, maybe we can revisit this in future when we need to reduce the size again 😄